docs/developer-docs/smart-contracts/overview/canister-lifecycle.mdx

```diff
@@ -0,0 +1,63 @@
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Canister lifecycle
+
+<MarkdownChipRow labels={["Beginner", "Concept"]} />
+
+## Write
+
+The ICP accepts and executes canister code in the WebAssembly (Wasm) binary format.
+Usually, developers write code in a higher-level language and then compile the code into a Wasm binary that can be deployed into a canister.
+
+## Deploy
+
+### Create
+
+A developer can create an empty canister on ICP by calling the system canister.
+It can be done using the `NNS frontend dapp` in the browser or `dfx` in the command line.
+The developer can specify the initial canister settings and choose the target subnet.
+Once the canister is created, the developer gets its canister ID and becomes the controller of the canister.
+
+<div class="text--center">
+<img src="/img/docs/create-canister.png" alt="Create canister" width="600"/>
+</div>
+
+### Install
+
+The next step after writing the code and creating a canister is installing the code.
+This can be done by calling the `install_code` endpoint of the system canister and passing the canister ID and the Wasm binary to it.
+The `dfx` tool has a command for making this call.
```

Comment:
Which command should this link to?

Reply:
This command: https://internetcomputer.org/docs/current/developer-docs/developer-tools/cli-tools/cli-reference/dfx-canister#dfx-canister-install

I'll link it.